src/polyfill.ts

```diff
@@ -16,28 +16,31 @@ import { LanguageService as VScodeLanguageService } from 'vscode/vs/editor/commo
-import { Configuration as MonacoConfiguration } from 'monaco-editor/esm/vs/platform/configuration/common/configurationModels.js'
-import { Configuration as VScodeConfiguration } from 'vscode/vs/platform/configuration/common/configurationModels.js'
+import { Configuration as MonacoConfiguration, ConfigurationModel as MonacoConfigurationModel, ConfigurationModelParser as MonacoConfigurationModelParser } from 'monaco-editor/esm/vs/platform/configuration/common/configurationModels.js'
+import { Configuration as VScodeConfiguration, ConfigurationModel as VScodeConfigurationModel, ConfigurationModelParser as VScodeConfigurationModelParser } from 'vscode/vs/platform/configuration/common/configurationModels.js'
 import { RawContextKey as MonacoRawContextKey, ContextKeyExpr as MonacoContextKeyExpr } from 'monaco-editor/esm/vs/platform/contextkey/common/contextkey.js'
 import { RawContextKey as VScodeRawContextKey, ContextKeyExpr as VScodeContextKeyExpr } from 'vscode/vs/platform/contextkey/common/contextkey.js'
 import { ErrorHandler as MonacoErrorHandler } from 'monaco-editor/esm/vs/base/common/errors.js'
 import { ErrorHandler as VScodeErrorHandler } from 'vscode/vs/base/common/errors.js'
 import { LanguagesRegistry as MonacoLanguagesRegistry } from 'monaco-editor/esm/vs/editor/common/services/languagesRegistry.js'
 import { LanguagesRegistry as VScodeLanguagesRegistry } from 'vscode/vs/editor/common/services/languagesRegistry.js'
-import { WorkspaceFolder as MonacoWorkspaceFolder } from 'monaco-editor/esm/vs/platform/workspace/common/workspace.js'
-import { IWorkspaceContextService, WorkbenchState, WorkspaceFolder as VScodeWorkspaceFolder } from 'vscode/vs/platform/workspace/common/workspace.js'
+import { WorkspaceFolder as MonacoWorkspaceFolder, Workspace as MonacoWorkspace } from 'monaco-editor/esm/vs/platform/workspace/common/workspace.js'
+import { IWorkspaceContextService, WorkbenchState, WorkspaceFolder as VScodeWorkspaceFolder, Workspace as VScodeWorkspace } from 'vscode/vs/platform/workspace/common/workspace.js'
 import { List as MonacoList } from 'monaco-editor/esm/vs/base/browser/ui/list/listWidget.js'
 import { List as VScodeList } from 'vscode/vs/base/browser/ui/list/listWidget.js'
 import { Color as MonacoColor, RGBA as MonacoRGBA } from 'monaco-editor/esm/vs/base/common/color.js'
 import { Color as VScodeColor, RGBA as VScodeRGBA } from 'vscode/vs/base/common/color.js'
 import { LogService as MonacoLogService } from 'monaco-editor/esm/vs/platform/log/common/logService.js'
 import { LogService as VScodeLogService } from 'vscode/vs/platform/log/common/logService.js'
+import { ConsoleLogger as MonacoConsoleLogger, MultiplexLogger as MonacoMultiplexLogger } from 'monaco-editor/esm/vs/platform/log/common/log'
+import { ConsoleLogger as VScodeConsoleLogger, MultiplexLogger as VScodeMultiplexLogger } from 'vscode/vs/platform/log/common/log'
```

Comment:
This must be log.js

Reply:
It breaks webpack build again

Reply:
Oops! here ce21dc83293d330318ce2ff64b577dbf8d843f92 to make sure it doesn't happen again